### DISCUSSION TITLE
Blender v4 support?

### DISCUSSION THREAD

**Discussion by gerroon** [nortikin/sverchok#4994] [General]
Hi

Does  anyone know if Sverchok will work with v4 sooninsh?

**Comment by Durman** [comment]
You can keep an eye on this https://github.com/nortikin/sverchok/pull/4983

**Comment by RRecoaro** [comment]
I hope I don't bother anyone, I'm not a programmer, it's possible that my question is stupid.
With blender version 4.0.2 and Sverchok version 1.3, the mesh is not displayed until before cooking.
What it means to do things blindly.
I tried version 3.6.9, and it doesn't have this problem. Everything works fine.
If anyone knows how to solve this problem for version 4.0.2, I would greatly appreciate it.
Best regards to all

**Comment by macdroid53** [comment]
FWIW I have corrected the following things and, at least the nodes I use do work.

 If you go to this file: scripts/addons/sverchok-master/nodes/script/sn_functor_b.py and comment out the following line: 
`import imp`
 the package can be used. The module imp has been deprecated for many Python revs and as of 3.12 has been removed and replaced with importlib. But, imp appears to not be used elsewhere in sverchok, so commenting it out seems to work. Since the import fails it prevents the addon from installing.

To get the stethoscope node working...this  line: 
`blf.size(font_id, int(text_height), 72)`

needs to change to:
`blf.size(font_id, int(text_height))`

in all occurrences. Specifically in this file: <your local config>/scripts/addons/sverchok-master/utils/sv_stethoscope_helper.py

It is also in:
./ui/bgl_callback_nodeview.py
./utils/nodeview_time_graph_drawing.py
./utils/sv_idx_viewer28_draw.py
